.github/workflows/docs.yml

```diff
@@ -0,0 +1,40 @@
+name: Build Docs
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
+    paths:
+      - './packages/appium/docs/**'
+      - './packages/docutils/**'
+      - './packages/typedoc-plugin-appium/**'
```

Comment:
we might in the future want to add docs dirs for plugins in the monorepo, but i don't think any exist yet

Reply:
I'm thinking separate workflows so docs build happens on per-package basis, and only if the pkg is modified